Selling items with level greater than +15

**Describe the bug**
When we try to sell an item that has its level greater than **+15** an error occur in the server and the client is freezing for some minutes.

**To Reproduce**
Steps to reproduce the behavior:
1. As a GM you need to drop an item with the level greater than +15, for example, "/item 9 4 18 1".
2. Since the max level is +15 so an **Excellent Bone Pants +2** will be dropped.
3. Try to sell this item in any NPC
4. The error below will occur in the server because of item.Level is +18 and the AdditionalDurabilityPerLevel only have *16* indexes ({ 0, 1, 2, 3, 4, 6, 8, 10, 12, 14, 17, 21, 26, 32, 39, 47 }).

_[ERROR] 2020-11-11 15:54:52,295 [MUnique.OpenMU.GameLogic.Player] [(null)] - Error while processing a message
System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at MUnique.OpenMU.GameLogic.ItemExtensions.GetMaximumDurabilityOfOnePiece(Item item) in C:\Projects\OpenMU\src\GameLogic\ItemExtensions.cs:line 72
   at MUnique.OpenMU.GameLogic.ItemPriceCalculator.CalculateBuyingPrice(Item item, Byte durability) in C:\Projects\OpenMU\src\GameLogic\ItemPriceCalculator.cs:line 449
   at MUnique.OpenMU.GameLogic.ItemPriceCalculator.CalculateBuyingPrice(Item item) in C:\Projects\OpenMU\src\GameLogic\ItemPriceCalculator.cs:line 316
   at MUnique.OpenMU.GameLogic.ItemPriceCalculator.CalculateSellingPrice(Item item) in C:\Projects\OpenMU\src\GameLogic\ItemPriceCalculator.cs:line 258
   at MUnique.OpenMU.GameLogic.PlayerActions.Items.SellItemToNpcAction.SellItem(Player player, Item item) in C:\Projects\OpenMU\src\GameLogic\PlayerActions\Items\SellItemToNpcAction.cs:line 62
   at MUnique.OpenMU.GameLogic.PlayerActions.Items.SellItemToNpcAction.SellItem(Player player, Byte slot) in C:\Projects\OpenMU\src\GameLogic\PlayerActions\Items\SellItemToNpcAction.cs:line 57
   at MUnique.OpenMU.GameServer.MessageHandler.Items.SellItemToNpcHandlerPlugIn.HandlePacket(Player player, Span`1 packet) in C:\Projects\OpenMU\src\GameServer\MessageHandler\Items\SellItemToNpcHandlerPlugIn.cs:line 33
   at MUnique.OpenMU.GameServer.MessageHandler.PacketHandlerPlugInContainer`1.HandlePacket(Player player, Span`1& packet, THandler handler) in C:\Projects\OpenMU\src\GameServer\MessageHandler\PacketHandlerPlugInContainer.cs:line 124
   at MUnique.OpenMU.GameServer.MessageHandler.PacketHandlerPlugInContainer`1.HandlePacket(RemotePlayer player, Span`1& packet) in C:\Projects\OpenMU\src\GameServer\MessageHandler\PacketHandlerPlugInContainer.cs:line 76
   at MUnique.OpenMU.GameServer.RemoteView.RemotePlayer.PacketReceived(ReadOnlySequence`1 sequence) in C:\Projects\OpenMU\src\GameServer\RemoteView\RemotePlayer.cs:line 143_

**Expected behavior**
The item.Level must be **2** instead of **18** or the server shouldn't allow dropping items greater than +15.

**Screenshots**
![ConsoleError](https://user-images.githubusercontent.com/942753/98828248-2015cc00-2438-11eb-91f3-0c5015dd9f7c.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome
 - Version [e.g. 22]

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6]
 - OS: [e.g. iOS8.1]
 - Browser [e.g. stock browser, safari]
 - Version [e.g. 22]

**Additional context**
Add any other context about the problem here.

--- a/src/GameLogic/PlugIns/ChatCommands/CreateItemChatCommandPlugIn.cs
+++ b/src/GameLogic/PlugIns/ChatCommands/CreateItemChatCommandPlugIn.cs
@@ -60,9 +60,15 @@ namespace MUnique.OpenMU.GameLogic.PlugIns.ChatCommands
         {
             var item = new TemporaryItem();
             var itemDefinition = player.GameContext.Configuration.Items.FirstOrDefault(def => def.Group == arguments.Group && def.Number == arguments.Number);
+
             if (itemDefinition == null)
             {
                 throw new ArgumentException($"[GM][/item] {arguments.Group} {arguments.Number} does not exists");
+            }
+
+            if (arguments.Level > itemDefinition.MaximumItemLevel)
+            {
+                throw new ArgumentException($"[GM][/item] Level cannot be greater than {itemDefinition.MaximumItemLevel}");
             }
 
             item.Definition = itemDefinition;

